pkg/providers/k8s/k8s.go

```diff
@@ -0,0 +1,136 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/helmfile/vals/pkg/api"
+	"github.com/helmfile/vals/pkg/log"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type provider struct {
+	log            *log.Logger
+	KubeConfigPath string
+	KubeContext    string
+}
+
+func New(l *log.Logger, cfg api.StaticConfig) *provider {
+	p := &provider{
+		log: l,
+	}
+
+	kubeConfig, err := getKubeConfig(cfg, *l)
+	if err != nil {
+		fmt.Printf("An error occurred getting the Kubeconfig path: %s", err)
```

Comment:
only print?

Reply:
Thanks! It's a good point. It made me realize some improvements in the error handling that I've now pushed.